hack/verify-codegen.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
not every platform has /bin/bash ;) /usr/bin/env bash is better, /bin/sh is the best (only if there are no bashisms, not sure).